src/qibotf/custom_operators/python/ops/qibo_tf_custom_operators.py

```diff
@@ -1,5 +1,15 @@
 """Use ops in python."""
 import tensorflow as tf
+from qibotf import __target_tf_version__, __version__
+if tf.__version__ < __target_tf_version__:  # pragma: no cover
+    raise RuntimeError(
+        f"qibotf {__version__} requires TensorFlow {__target_tf_version__}."
+        "Please upgrade TensorFlow or downgrade qibotf.")
+elif tf.__version__ > __target_tf_version__:  # pragma: no cover
+    raise RuntimeError(
+        f"qibotf {__version__} requires TensorFlow {__target_tf_version__}."
+        "Please upgrade qibotf or downgrade TensorFlow.")
```

Comment:
I understand that this checks that versions are in agreement with the compatibility matrix from the README, however isn't this a bit more restrictive than necessary? For example currently, qibotf custom operators work with both tf2.4 and tf2.5, however this check will restrict someone from using the latest qibotf with tf2.4. Is there any reason we would like to restrict that since there are no changes from the custom operator side?